cms/templates/asset_index.html

```diff
@@ -13,10 +13,7 @@
 <%namespace name='static' file='static_content.html'/>
 
 <%block name="header_extras">
-% if waffle_flag_enabled:
-    <link rel="stylesheet" type="text/css" href="${static.url('common/css/vendor/studio-frontend.min.css')}" />
-    <script type="text/javascript" src="${static.url('common/js/vendor/assets.min.js')}" defer></script>
```

Comment:
This is part of why I was having so many problems - these minified assets are never touched by webpack.